data/human/human missions.txt

```diff
@@ -5547,3 +5547,130 @@ mission "Quicksilver Mixup 1"
 			
 
 event "quicksilver mixup 2 available"
+
+
+
+mission "Argosy Hijacking (Trigger)"
+	invisible
+	source
+		government "Syndicate"
+	to offer
+		random < 10
+	on offer
+		event "argosy hijacking trigger"
+		fail
+
+event "argosy hijacking trigger"
+
+mission "Argosy Hijacking"
+	landing
+	name "Stop Ship Hijacking"
+	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not board, capture, or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
+	source
+		government "Syndicate"
+		not attributes "station"
+		near "Alcyone" 4
+		near "Almach" 3
+		near "Gienah" 3
```

Comment:
```suggestion
		not near "Sol" 3
```
Since warp_core mentioned how we can't use `or` for location filters, the next closest thing is probably having it be a Syndicate world away from Sol. Not an exact fit, but it's close.